src/main/java/net/minecraftforge/world/WorldCapabilityData.java

```diff
@@ -0,0 +1,58 @@
+package net.minecraftforge.world;
```

Comment:
Instead of creating a whole new package and all that implies, for one class, just shove it in common.util.